### PR TITLE
Ccg software providers 2 #16224 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/SUSE/spacewalk/issues/16224 - **where you can find links to the generated HTML output (for reviewing)**

This PR supersedes https://github.com/uyuni-project/uyuni-docs/pull/1656

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/clients-centos.adoc [hunk 1]
```diff
@@ -3,6 +3,19 @@
...
 This section contains information about registering Salt clients running {centos} operating systems.
 
+ifeval::[{suma-content} == true]
+// Use list format: repository URLs / packages and metadata / version info
+// Do we need to provide special info for uyuni users?
+[IMPORTANT]
+====
+* {centos} repository URLs are available from {scc}
++
+// Check the following statement!
```
Comment (lucidd) [inline modules/client-configuration/pages/clients-centos.adoc:13]: this should be correct as centos comes form official centos urls

**Comment by lucidd** [comment]
it looks correct so far. As a reference you can take a look at the output of `curl https://scc.suse.com/suma/product_tree.json | jq '.[].url'` to see the url we are using. If it points to an external url like centos.org or ubuntu.com then the packages doesn't come from us.

**Review by lucidd — APPROVED** [review]

**Review by 0rnela — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]
LGTM, thank you Karl.

**Comment by keichwa** [comment]
> https://scc.suse.com/suma/product_tree.json

Thanks for all your help --this is valuable info!

**Pull request merged**